### DISCUSSION TITLE
Please Help!  ModuleNotFoundError: No module named 'mmdet.ops'

### DISCUSSION THREAD

**Discussion by pear313063** [open-mmlab/mmdetection#10023] [Q&A]
When I entered these two, then error, how to solve it?

from mmdet.ops.nms import nms_wrapper
ModuleNotFoundError: No module named 'mmdet.ops'

from mmdet.ops.bbox import cython_bbox
ModuleNotFoundError: No module named 'mmdet.ops'

**Comment by hhaAndroid** [comment]
@pear313063 Your version is really too old, can you update the version?

**Reply by pear313063** [reply]
Sorry, which version is too old?
